lib/openstudio-standards/standards/Standards.AirLoopHVAC.rb

```diff
@@ -1928,6 +1928,10 @@ def air_loop_hvac_apply_minimum_vav_damper_positions(air_loop_hvac, has_ddc = tr
           zone_oa = thermal_zone_outdoor_airflow_rate(zone)
           vav_terminal = equip.to_AirTerminalSingleDuctVAVReheat.get
           air_terminal_single_duct_vav_reheat_apply_minimum_damper_position(vav_terminal, zone_oa, has_ddc)
+        elsif equip.to_AirTerminalSingleDuctParallelPIUReheat.is_initialized
+          zone_oa = thermal_zone_outdoor_airflow_rate(zone)
+          fp_vav_terminal = equip.to_AirTerminalSingleDuctParallelPIUReheat.get
+          air_terminal_single_duct_parallel_piu_reheat_apply_minimum_primary_airflow_fraction(fp_vav_terminal, zone_oa)
```

Comment:
The minimum primary air flow fraction was set during the FPB fan power calculation, the assignment was moved here for consistency.